### ISSUE TITLE
Application crash - Font 'Consolas' does not support style 'Regular' in Windows Store version v2.2.3 in some environments

### ISSUE THREAD

**Issue by PawelTroka** [PawelTroka/Computator.NET#99]
This is really similar to issue #88 and #84. It happens after upgrade to Windows 10 Creators Update.

Log:
```
2017-11-06 22:43:11.43.003 [3]ERROR System.Drawing.Font.CreateNativeFont - Unhandled UI Exceptionis terminting: True, System.ArgumentException: Font 'Consolas' does not support style 'Regular'.
   at System.Drawing.Font.CreateNativeFont()
   at System.Drawing.Font.Initialize(FontFamily family, Single emSize, FontStyle style, GraphicsUnit unit, Byte gdiCharSet, Boolean gdiVerticalFont)
   at System.Drawing.Font..ctor(String familyName, Single emSize)
   at Computator.NET.Dialogs.LoadingScreen..ctor()
   at Computator.NET.Dialogs.LoadingScreen.ShowForm()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart() System.ArgumentException: Font 'Consolas' does not support style 'Regular'.
   at System.Drawing.Font.CreateNativeFont()
   at System.Drawing.Font.Initialize(FontFamily family, Single emSize, FontStyle style, GraphicsUnit unit, Byte gdiCharSet, Boolean gdiVerticalFont)
   at System.Drawing.Font..ctor(String familyName, Single emSize)
   at Computator.NET.Dialogs.LoadingScreen..ctor()
   at Computator.NET.Dialogs.LoadingScreen.ShowForm()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart() 

```

**Comment by PawelTroka** [comment]
This might be related:
1. https://social.technet.microsoft.com/Forums/windows/en-US/4a1b0d78-bed3-465b-be38-859b4180a01b/missing-fonts-after-creators-update-installed?forum=win10itprogeneral
2. https://answers.microsoft.com/en-us/windows/forum/windows_10-start/some-fonts-are-missing-after-upgrade/95839dfa-0df2-4bc0-875a-fd6b57e61fe4
3. https://docs.microsoft.com/en-us/windows/threat-protection/block-untrusted-fonts-in-enterprise

**Comment by PawelTroka** [comment]
Should be solved by 9a1096e99c3ed79a394e7dc82eb5b2dabea242bd
